src/Microsoft.VisualStudio.AppDesigner/Microsoft.VisualStudio.AppDesigner.vbproj

```diff
@@ -24,7 +24,6 @@
     <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
     <TargetFrameworkProfile />
     <ProjectSystemLayer>VisualStudioDesigner</ProjectSystemLayer>
-    <StrongNameCertificate>Microsoft</StrongNameCertificate>
```

Comment:
If I'm reading correctly, this is now being signed with MicrosoftShared - which is wrong.